deps/pylib.sh

```diff
@@ -1 +1,2 @@
 pip install requests_toolbelt filemagic ftputil
+pip install git+https://github.com/metabolite-atlas/metatlas
```

Comment:
Or are the runtime and the narrative distinct environments?



—
Sent from Mailbox

On Fri, May 8, 2015 at 5:18 PM, Roman Sutormin <notifications@github.com>
wrote:

>> @@ -1 +1,2 @@
>>  pip install requests_toolbelt filemagic ftputil
>> +pip install git+https://github.com/metabolite-atlas/metatlas
> It doesn't matter. We need to have some temporary instructions until hdf5 is added into kbase runtime (which could take long time actually). It means there should be script in "transform/deps" folder installing it manually. I'll help you with it. What version of hdf5 do you prefer to use? If this script doesn't require sudo privileges then you just run it from make deploy target. If it does (like apt-get) then you mention about this script in readme file as preinstalled requirement. It's better to do it ones and make sure other people won't struggle with it hating you.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/kbase/transform/pull/151/files#r29981531